lib/resourcedelete/apiext.go

```diff
@@ -0,0 +1,41 @@
+package resourcedelete
```

Comment:
Did we want to auto-generate any of this in [`generate-lib-resource`][1]?  Or do these need enough hand-touching to make them annoying to automate?

[1]: https://github.com/openshift/cluster-version-operator/blob/7e7544097149708190903b1f4d088a4529704886/hack/generate-lib-resources.py#L7-L8